09_qgep_dictionaries.sql

```diff
@@ -485,6 +485,8 @@ CREATE TABLE qgep_sys.dictionary_od_field (
+ INSERT INTO qgep_sys.dictionary_od_field (class_id, attribute_id, table_name, field_name, field_name_en, field_name_de, field_name_fr, field_name_it, field_name_ro, field_description_en, field_description_de, field_description_fr, field_description_it, field_description_ro, field_mandatory, field_visible, field_datatype, field_unit_en, field_unit_description_en, field_unit_de, field_unit_description_de, field_unit_fr, field_unit_description_fr, field_unit_it, field_unit_description_it, field_unit_ro, field_unit_description_ro, field_max, field_min) VALUES (31,9999999,'reach','swmm_default_coefficient_of_friction','swmm_default_coefficient_of_friction',NULL,NULL,NULL,NULL,'Default coefficient of friction for SWMM export.',NULL,NULL,NULL,NULL,NULL,'true','smallint','[m^(1/3)/s]','Manning-Strickler [m^(1/3)/s]','[m^(1/3)/s]','Manning-Strickler [m^(1/3)/s]','','','','','','',0,999);
+ INSERT INTO qgep_sys.dictionary_od_field (class_id, attribute_id, table_name, field_name, field_name_en, field_name_de, field_name_fr, field_name_it, field_name_ro, field_description_en, field_description_de, field_description_fr, field_description_it, field_description_ro, field_mandatory, field_visible, field_datatype, field_unit_en, field_unit_description_en, field_unit_de, field_unit_description_de, field_unit_fr, field_unit_description_fr, field_unit_it, field_unit_description_it, field_unit_ro, field_unit_description_ro, field_max, field_min) VALUES (31,9999999,'reach','hydraulic_load','hydraulic_load',NULL,NULL,NULL,NULL,'Dimensionierungsabfluss geteilt durch Normalabflusskapazität der Leitung [%].','Dimensionierungsabfluss geteilt durch Normalabflusskapazität der Leitung [%].','Débit de dimensionnement divisé par la capacité d''écoulement normale de la conduite [%].',NULL,NULL,NULL,'true','decimal(7,3)','[%]','Dimensionierungsabfluss geteilt durch Normalabflusskapazität der Leitung [%].','[m^(1/3)/s]','Dimensionierungsabfluss geteilt durch Normalabflusskapazität der Leitung [%].','','','','','','',0,100);
```

Comment:
And adapt the attributename here and add the English description.

Reply:
Done